### ISSUE TITLE
error while creating first snapshot

### ISSUE THREAD

**Issue by shabazpatel** [datmo/datmo#114]
```
ubuntu@ip-172-31-24-161:~/keras-fashion-mnist$ datmo snapshot create -m "creating my first snapshot"
Creating a new snapshot
An exception occurred: Commit ref given does not match a git commit within the tree: 992a942e2a3393922991907e222440f6b9a0c973
```

**Comment by asampat3090** [comment]
seems like this has been fixed once a user has created a file, however it seems like an anomalous line "fatal: your current branch 'master' does not have any commits" is still shown in the output. see below

```
Creating a new snapshot
fatal: your current branch 'master' does not have any commits yet
INFO: Successfully saved requirements file in /Users/anandsampat/Downloads/datmotest/requirements.txt
Created snapshot with id: ba6d7dd5455c47055b9d6f788954a96e80bb7cf1
```

**Comment by asampat3090** [comment]
9312d9d : all stderr and stdout for git code driver is piped to outputs that can be manipulated in the code using subprocess.Popen. the errors from the subprocesses are only displayed if we raise an error. we can change that functionality as needed.

**Comment by asampat3090** [comment]
fixed with PR #115
